src/Datadog.Trace/Agent/ZipkinApi.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Agent
+{
+    internal class ZipkinApi : IApi
+    {
+        private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.For<ZipkinApi>();
+
+        private readonly TracerSettings _settings;
+        private Uri _tracesEndpoint;
+
+        public ZipkinApi(TracerSettings settings)
+        {
+            Log.Debug("Creating new Zipkin Api");
+
+            _settings = settings ?? throw new ArgumentNullException(nameof(settings));
+            _tracesEndpoint = _settings.AgentUri; // User needs to include the proper path.
+        }
```

Comment:
The `Api` implementation has some logging here about the framework it seems that we should perhaps move it to some higher level so it gets logged independently of the exporter.